gui/.gitignore

```diff
@@ -0,0 +1,17 @@
+
+# Default Dependency directories of a NodeJS project.
+node_modules
+
+# Decoding output directory of the APKs. The application uses this dir
+# to send the decode results there.
+output
+
+# Internal Library directoy of the project, where the
```

Comment:
typo in _ directoy_

Reply:
Could you please add an example what you suggesting here? I didn't understand what must do here.